Tests/SWBTaskExecutionTests/InfoPlistProcessorTaskTests.swift

```diff
@@ -1582,7 +1582,7 @@ fileprivate struct InfoPlistProcessorTaskTests: CoreBasedTests {
         let platform = try #require(core.platformRegistry.lookup(name: platformName), "invalid platform name '\(platformName)'")
         let executionDelegate = MockExecutionDelegate(core: try await getCore())
 
-        let action = InfoPlistProcessorTaskAction(try prepareContext(InfoPlistProcessorTaskActionContext(scope: scope, productType: nil, platform: platform, sdk: try #require(core.sdkRegistry.lookup(platformName)), sdkVariant: nil, cleanupRequiredArchitectures: []), fs: executionDelegate.fs))
```

Comment:
This and a few similar examples should be valid - `try #require(core.sdkRegistry.lookup(platformName))` looks up an SDK, assert that it is non-nil, and then passes it to the InfoPlistProcessorTaskActionContext via a T-to-T? conversion. I'm also not seeing this warn locally - perhaps the implementation of the #require macro has changed?

I think I'm ok taking this change anyway, since a nil SDK here should cause downstream failures, they just won't be quite as close to where the problem occurred

Reply:
The compiler was pointing out that the return type doesn't indicate optionality and so the require is unnecessary.

Reply:
Huh, `lookup` is defined as `package func lookup(_ name: String) -> SDK?`. But I'm fine with landing this